src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs

```diff
@@ -24,7 +24,7 @@ internal partial class InstallDialog : DialogWindow
 
         public InstallDialog(IDependencies dependencies, ILibraryCommandService libraryCommandService, string configFileName, string fullPath, string rootFolder)
         {
-            InstallationFolder.DestinationFolder = fullPath.Replace(rootFolder, "").Replace('\\', '/');
```

Comment:
>Replace [](start = 60, length = 7)

Use StringComparison.OrdinalIgnoreCase here

Reply:
use starswith for it

---
In reply to: [202842715](https://github.com/aspnet/LibraryManager/pull/316#discussion_r202842715) [](ancestors = 202842715)